sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationChecker.scala

```diff
@@ -60,6 +68,14 @@ object UnsupportedOperationChecker {
           "streaming DataFrames/Datasets")(plan)
     }
 
+    // Disallow multiple streaming deduplications
```

Comment:
we should support these. Example use case:
I dedup on some higher level columns to gain exactly once semantics (infrastructure/application specific). Then I do data transformations, then I do a dedup on some more specific data, e.g. region (query specific)